keystore/v2/keystore/keyStore.go

```diff
@@ -99,6 +108,58 @@ func DescribeKeyRings(keyRings []string, keyStore api.KeyStore) ([]keystore.KeyD
 	return keys, nil
 }
 
+// DescribeHistoricalKeyRings describes multiple key rings by their purpose paths.
+func DescribeHistoricalKeyRings(keyRings []string, keyStore api.KeyStore) ([]keystore.KeyDescription, error) {
+	keys := make([]keystore.KeyDescription, 0, len(keyRings))
+	for i := range keyRings {
+		descriptions, err := keyStore.DescribeHistoricalKeyRing(keyRings[i])
+		if err != nil {
+			return nil, err
+		}
+		keys = append(keys, descriptions...)
+	}
+	return keys, nil
+}
+
+// DescribeHistoricalKeyRing describes key ring by its purpose path.
+func (s *ServerKeyStore) DescribeHistoricalKeyRing(path string) ([]keystore.KeyDescription, error) {
+	if path == poisonKeyPath {
+		return s.listHistoricalRings(path, poisonKeyPath, []byte{})
+	}
+
+	if path == auditLogSymmetricKeyPath {
+		return s.listHistoricalRings(path, PurposePoisonSym, []byte{})
+	}
+	if path == poisonSymmetricKeyPath {
+		return s.listHistoricalRings(path, poisonSymmetricKeyPath, []byte{})
+	}
+
+	// Paths which are not server-global symmetric keys look like this:
+	//
+	//     client/${client_id}/storage
+	//
+	// And transport paths look like this, with an additional component:
```

Comment:
now we don't have transport paths and we expected 3 parts of the path below. So, lets update comment to actual the state